src/main/java/tr/com/srdc/cda2fhir/transform/CCDTransformerImpl.java

```diff
@@ -304,8 +304,8 @@ private ICDASection findCDASection(Section section) {
 	}
 
 	/**
-	 * Transforms a Consolidated CDA (C-CDA) 2.1 Continuity of Care Document (CCD)
-	 * instance to a Bundle of corresponding FHIR resources
+	 * <<<<<<< HEAD Transforms a Consolidated CDA (C-CDA) 2.1 Continuity of Care
+	 * Document (CCD) instance to a Bundle of corresponding FHIR resources
 	 *
```

Comment:
Can you please fix this? Looks like a leftover from conflict resolution.

Reply:
Will do, sorry.